pkg/systemd/services/timer-dialog.jsx

```diff
@@ -85,6 +87,11 @@ const CreateTimerDialogBody = ({ owner }) => {
         validationFailed.command = true;
     if (!/^[0-9]+$/.test(delayNumber))
         validationFailed.delayNumber = true;
+    if (custom) {
+        cockpit.spawn(["systemd-analyze", "calendar", custom])
+                .then(() => setValidateCustom(false))
+                .catch(() => setValidateCustom(true));
+    }
```

Comment:
we run this waay to often but show it only when you click on submit button. Let's dial that down then

Reply:
I did find it interesting that the validation happens on the fields in the background but only become displayed after the initial submit.  After that all fields are validated "realtime".  should we change this for all inputs?  I kind of liked the ability to validate the custom input as I was writing it but it does become resource heavy.

Reply:
Right, I think that is more bug than feature :smile: 
I think what we want is to validate changed fields on focus out and then do validation on submit (to catch missing empty fields).

Do you want to take a look at that? Just please pull the changes I pushed here and feel free to hack away. If not I am happy to take a look at it.

Reply:
I can take a look but it wont likely be until Friday with my schedule.  if you get to it before then so be it.  ill check back here before I start anything.

Reply:
@marusak I have made numerous updates on how and when inputs are validated.  Please take a look and let me know.  I did leave some comments in the code for now just to assist in checking.  I can remove later if all is good.